src/styles/index.css

```diff
@@ -0,0 +1,114 @@
+/* stylelint-disable rule-empty-line-before */
+body {
+  margin: 0;
+}
+
+html {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(7 * 100px + 6px);
+  padding: 10px;
+  gap: 1px;
+  counter-reset: day-counter 0;
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  background-color: #eee;
+  border: 1px solid black;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.calendar__day::before {
+  counter-increment: day-counter;
+  content: counter(day-counter);
+}
+
+.calendar__day:hover {
+  cursor: pointer;
+  background-color: #ffbfcb;
+  transform: translateY(-20px);
+  transition: all 0.5s;
+}
+
+.calendar--start-day-mon .calendar__day:nth-child(1) {
+  margin-left: calc(0 * (100px + 1px));
+}
+
+.calendar--start-day-tue .calendar__day:nth-child(1) {
+  margin-left: calc(1 * (100px + 1px));
+}
+
+.calendar--start-day-wed .calendar__day:nth-child(1) {
+  margin-left: calc(2 * (100px + 1px));
+}
+
+.calendar--start-day-thu .calendar__day:nth-child(1) {
+  margin-left: calc(3 * (100px + 1px));
+}
+
+.calendar--start-day-fri .calendar__day:nth-child(1) {
+  margin-left: calc(4 * (100px + 1px));
+}
+
+.calendar--start-day-sat .calendar__day:nth-child(1) {
+  margin-left: calc(5 * (100px + 1px));
+}
+
+.calendar--start-day-sun .calendar__day:nth-child(1) {
+  margin-left: calc(6 * (100px + 1px));
+}
+.calendar--month-length-28 .calendar__day:nth-child(n + 29) {
+  display: none;
+}
+.calendar--month-length-29 .calendar__day:nth-child(n + 30) {
+  display: none;
+}
+.calendar--month-length-30 .calendar__day:nth-child(n + 31) {
+  display: none;
+}
+.calendar--month-length-31 .calendar__day:nth-child(n + 32) {
+  display: none;
+}
+
+.calendar--start-day--mon .calendar__day:first-child {
+  margin-left: 1px;
+}
+
+.calendar--start-day--tue .calendar__day:first-child {
+  margin-left: 101px;
+}
+
+.calendar--start-day--wed .calendar__day:first-child {
+  margin-left: 202px;
+}
+
+.calendar--start-day--thu .calendar__day:first-child {
+  margin-left: 303px;
+}
+
+.calendar--start-day--fri .calendar__day:first-child {
+  margin-left: 404px;
+}
+
+.calendar--start-day--sat .calendar__day:first-child {
+  margin-left: 505px;
+}
+
+.calendar--start-day--sun .calendar__day:first-child {
+  margin-left: 606px;
```

Comment:
These styles are duplicates of the ones above (lines 48-74) for setting the margin-left of the first calendar day. Consider removing these duplicates to avoid redundancy and potential confusion.